src/TodoForm.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import useInputState from "./useInputState";
+import "./styles.css";
```

Comment:
Why are you importing the styles here?
Are they for the whole application or for this single component?

Reply:
don't know, but changed it.